src/widgets/category-ingredient-container/category-ingredient-container.ui.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentProps } from 'react';
+
+import { IngredientCategory } from '@/entities/ingredient.type';
+import { IngredientCategoryCard } from '@/features/ingredient-category-card/ingredient-category-card.ui';
+
+import { INGREDIENT_CATEGORIES } from './category-ingredient-container.config';
+
+interface IngredientCategoryContainerProps extends ComponentProps<'div'> {
+  ingredientCategories?: readonly IngredientCategory[];
+}
+
+export function IngredientCategoryContainer({
+  ingredientCategories = INGREDIENT_CATEGORIES,
+  ...props
+}: IngredientCategoryContainerProps) {
+  return (
+    <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4" {...props}>
+      {ingredientCategories.map((ingredientCategory) => (
+        <div key={ingredientCategory.title} className="flex justify-center">
+          <IngredientCategoryCard
+            ingredientImageSrc={ingredientCategory.imageSrc}
+            ingredientIcon={ingredientCategory.icon}
+            ingredientTitle={ingredientCategory.title}
+            ingredientDescription={ingredientCategory.description}
+            className="w-full max-w-[326px]"
```

Comment:
직전의 리뷰들에 의해 이 부분은 다음과 같이 수정하면 될 것 같습니다.
```suggestion
          <IngredientCategoryCard
            {...category}
            className="w-full max-w-[326px]"
```